addons/account/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,387 @@
+account      / account.invoice          / vendor_bill_id (many2one)     : NEW relation: account.invoice
+account      / account.invoice          / vendor_display_name (char)    : NEW
+# NOTHING TO DO
+
+account      / account.invoice.line     / display_type (selection)      : NEW selection_keys: ['line_note', 'line_section']
+# DONE: pre-migration: add a row for each section
+
+account      / account.invoice.line     / uom_id (many2one)             : relation is now 'uom.uom' ('product.uom')
+# NOTHING TO DO
+
+account      / account.invoice.tax      / analytic_tag_ids (many2many)  : NEW relation: account.analytic.tag
+# NOTHING TO DO
+
+account      / account.journal          / alias_id (many2one)           : NEW relation: mail.alias
+# DONE: post-migration: filled for all journals fo type 'purchase'
+
+account      / account.journal          / bank_statements_source (selection): selection_keys is now 'function' ('['manual', 'undefined']')
+# DONE: post-migration: mapped 'manual' value to 'undefined'
+
+account      / account.journal          / post_at_bank_rec (boolean)    : NEW
+# NOTHING TO DO
+
+account      / account.move             / auto_reverse (boolean)        : NEW
+account      / account.move             / reverse_date (date)           : NEW 
+account      / account.move             / reverse_entry_id (many2one)   : NEW relation: account.move
+account      / account.move             / tax_type_domain (char)        : NEW 
+# DONE: pre-migration: renames in case account_reversal module was installed
+# DONE: post-migration: filled if not account_reversal module was installed
+# DONE: post-migration: filled reverse_date
+
+account      / account.move.line        / product_uom_id (many2one)     : relation is now 'uom.uom' ('product.uom')
+account      / account.move.line        / recompute_tax_line (boolean)  : NEW 
+account      / account.move.line        / tax_line_grouping_key (char)  : NEW 
+# NOTHING TO DO: new optional content
+
+account      / account.payment          / company_id (many2one)         : not stored anymore
+account      / account.payment          / message_last_post (datetime)  : DEL
+account      / account.payment          / message_main_attachment_id (many2one): NEW relation: ir.attachment
+# NOTHING TO DO
+
+account      / account.payment          / multi (boolean)               : NEW
+# NOTHING TO DO
+
+account      / account.payment          / partner_bank_account_id (many2one): NEW relation: res.partner.bank
+# NOTHING TO DO: Odoo makes you a proposal in the form, but not forces you to put a bank account unless if exists a payment method that forces it. But in odoo community there doesn't exist a payment method of this kind.
+
+account      / account.payment.term.line / day_of_the_month (integer)    : NEW
+# NOTHING TO DO: new optional technical value, only used for in the day_after_invoice_date option
```

Comment:
`day_after_invoice_date` option already existed in v11, and in that case `days` field was used, so you have to transfer that value in that case.

Reply:
I don't undertand what you say. In v11, in `compute` method the code was:
```python
        if line.option == 'day_after_invoice_date':
            next_date += relativedelta(days=line.days)
```
and v12 has:
```python
        if line.option == 'day_after_invoice_date':
            next_date += relativedelta(days=line.days)
            if line.day_of_the_month > 0:
                months_delta = (line.day_of_the_month < next_date.day) and 1 or 0
                next_date += relativedelta(day=line.day_of_the_month, months=months_delta)
```
The `day_of_the_month` is a manual introduced value. As it didn't exist in v11, then in v12 I think I shouldn't fill it. I don't see why would we set `day_of_the_month = days`.

Reply:
OK, got it